core/dbt/config/runtime.py

```diff
@@ -549,9 +550,9 @@ def from_args(cls: Type[RuntimeConfig], args: Any) -> 'RuntimeConfig':
 
 
 UNUSED_RESOURCE_CONFIGURATION_PATH_MESSAGE = """\
-WARNING: Configuration paths exist in your dbt_project.yml file which do not \
-apply to any resources.
-There are {} unused configuration paths:\n{}
+Configuration paths exist in your dbt_project.yml file which do not \
+apply to any resources. 
+\t     There are {} unused configuration paths:\n{}
```

Comment:
Instead of that `\n`, this should really just be a regular newline!. Also, I suggest using four spaces rather than 5 spaces and a hard tab, that indent is huge
```suggestion
    There are {} unused configuration paths:
{}
```

Reply:
Took out `\n` here and removed the formatting from this - I think that it doesn't really add to the story here and I should have focused more on just the warning tag. It's something that can definitely be done later!